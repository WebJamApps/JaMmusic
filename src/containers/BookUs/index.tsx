import { Button } from '@mui/material';
import { Inquiry } from '../NewHomepage/Inquiry';

const songlistUrl = 'https://docs.google.com/spreadsheets/d/1uePCuHO6qxg2tHTqGist_O4iYXJjy9AgSmXvMgF9NPM/edit?usp=sharing';
export function BookUs() {
  return (
    <div className="bookus" style={{ margin: 'auto', padding: '6px', paddingRight: '12px' }}>
      <div style={{ maxWidth: '800px' }}>
        <h3 style={{ textAlign: 'center' }}>Book Us</h3>
        <h4 style={{ fontWeight: 'normal' }}>
          We are always looking for regional performance opportunites,
          and would consider traveling even further away from our home in Salem, VA for just the right situation.
          <Button
            style={{ marginLeft: '8px' }}
            size="small"
            variant="contained"
            onClick={() => window.open(songlistUrl)}
          >
            Current Songlist
          </Button>

        </h4>
        <p style={{ textAlign: 'center' }}>
          Please fill out this form and we will get back with you soon.
        </p>
        <Inquiry country="United States" hideTitle />
      </div>
    </div>
  );
}
