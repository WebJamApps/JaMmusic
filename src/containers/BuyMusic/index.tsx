import { Component } from 'react';
import Jsb from './BuyLinks/joshShermanBand';
import Jss from './BuyLinks/joshShermanSolo';
import Jsyt from './MediaLinks/joshShermanYoutube';
import Jssp from './MediaLinks/joshShermanSpotify';
import Jsam from './MediaLinks/joshShermanApple';
import commonUtils from '../../lib/utils';

export default class BuyMusic extends Component {
  commonUtils: { setTitleAndScroll: (pageTitle: string, width: number) => void };

  constructor(props: Record<string, unknown>) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Buy Music', window.screen.width); }

  render(): JSX.Element {
    return (
      <div
        className="page-content"
        style={{
          paddingRight: '15px', maxWidth: '800px', margin: 'auto', borderRight: 'none',
        }}
      >
        <h2 style={{
          textAlign: 'center', margin: '20px', fontWeight: 'bold', marginBottom: '10px',
        }}
        >
          Buy from CD Baby
        </h2>
        <div className="row top-row-style">
          <Jsb />
          <Jss />
        </div>
        <Jssp />
        <Jsyt />
        <Jsam />
      </div>
    );
  }
}
