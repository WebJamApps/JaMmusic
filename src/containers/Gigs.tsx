import React, { useContext, useState } from 'react';
import { GigsContext } from '../providers/Gigs.provider';

export const Gigs = (): JSX.Element => {
  const { gigs } = useContext(GigsContext);
  console.log(gigs);
  const [allGigs, setGigs] = useState(gigs);
  return (
    <div style={{ margin: 'auto' }}>
      <h3>Gigs</h3>
      {JSON.stringify(allGigs)}
    </div>
  );
};
