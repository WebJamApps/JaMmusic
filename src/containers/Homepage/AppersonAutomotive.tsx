import React from 'react';

const AppersonAutomotive = (): JSX.Element => (
  <div className="elevation2 project apperson">
    <h4>Apperson Automotive</h4>
    <div>
      <img
        alt="Apperson Automotive Home Page"
        src="../static/imgs/AppersonAutomotive.png"
      />
      <br />
      <a
        href="https://www.appersonautomotive.com"
        target="_blank"
        rel="noreferrer noopener"
      >
        appersonautomotive.com
      </a>
    </div>
  </div>
);

export default AppersonAutomotive;
