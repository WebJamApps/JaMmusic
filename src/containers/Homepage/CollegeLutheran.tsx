import React from 'react';

const CollegeLutheran = (): JSX.Element => (
  <div className="elevation2 project" style={{ minWidth: '313px' }}>
    <h4>College Lutheran Church</h4>
    <img
      alt="College Lutheran Church Homepage"
      src="https://dl.dropboxusercontent.com/s/354maifx0dkzt9s/Screenshot%20from%202019-02-22%2016-01-50.png?dl=0"
    />
    <br />
    <a
      href="https://www.collegelutheran.org"
      target="_blank"
      rel="noreferrer noopener"
    >
      Collegelutheran.org
    </a>
  </div>
);

export default CollegeLutheran;
