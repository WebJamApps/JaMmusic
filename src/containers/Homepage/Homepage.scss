.MuiBox-root css-otmogo {
    padding-top: 0px;
}