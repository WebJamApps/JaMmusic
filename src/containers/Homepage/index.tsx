import React, { Component } from 'react';
import ReactResizeDetector from 'react-resize-detector';
import WideAboutUs from './Widescreen/WideAbout';
import WideCurrentProjects from './Widescreen/WideCurrentProjects';
import NarrowAboutUs from './Narrowscreen/NarrowAbout';
import NarrowCurrentProjects from './Narrowscreen/NarrowCurrentProjects';
import FacebookFeed from './Narrowscreen/NarrowFacebookFeed';
import Inquiry from '../../components/inquiry';
import commonUtils from '../../lib/commonUtils';

interface HomepageState { width: number }

export default class Home extends Component<{}, HomepageState> {
  parentRef: any;

  commonUtils: any;

  constructor(props: any) {
    super(props);
    this.parentRef = React.createRef();
    this.onResize = this.onResize.bind(this);
    this.state = { width: 320 };
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('', window.screen.width); }

  onResize(width: number) {
    this.setState({ width });
  }

  render() {
    const { width } = this.state;
    return (
      <div>
        {width >= 1004
          ? (
            <div className="page-content">
              <div className="anchor"> </div>
              <WideAboutUs />
              <hr />
              <WideCurrentProjects />
              <p style={{ fontSize: '6pt', marginBottom: '0' }}>&nbsp;</p>
            </div>
          )
          : (
            <div className="page-content">
              <NarrowAboutUs />
              <hr />
              <NarrowCurrentProjects />
              <p style={{ fontSize: '6pt', marginBottom: '0' }}>&nbsp;</p>
              <FacebookFeed />
              <p style={{ fontSize: '6pt', marginBottom: '0' }}>&nbsp;</p>
              <Inquiry />
              <p style={{ fontSize: '6pt', marginBottom: '0' }}>&nbsp;</p>
            </div>
          )}
        <ReactResizeDetector handleWidth handleHeight onResize={this.onResize} targetDomEl={this.parentRef.current} />
      </div>
    );
  }
}
