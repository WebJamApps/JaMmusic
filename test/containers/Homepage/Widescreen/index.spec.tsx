import renderer from 'react-test-renderer';
import { WideOrNarrow, Homepage } from 'src/containers/Homepage';

describe('Homepage', () => {
  it('renders Homepage', () => {
    const homepage = renderer.create(<Homepage />).toJSON();
    expect(homepage).toMatchSnapshot();
  });
  it('renders WideAboutUs', () => {
    const wideOrNarrow:any = renderer.create(<WideOrNarrow width={1005} />).toJSON();
    expect(wideOrNarrow.props.className.includes('wideHome')).toBe(true);
  });
});
