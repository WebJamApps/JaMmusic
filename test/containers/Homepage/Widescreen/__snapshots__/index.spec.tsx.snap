// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders Homepage 1`] = `
<div>
  <div
    className="page-content narrowHome"
  >
    <div
      className="notWidescreen"
      style={
        {
          "marginTop": "1px",
        }
      }
    >
      <div
        id="slideshow"
      >
        <img
          alt="Web Jam Banner"
          src="../static/imgs/banner1.jpg"
        />
      </div>
      <div
        style={
          {
            "marginBottom": "-20px",
            "padding": "40px",
          }
        }
      >
        <p
          style={
            {
              "fontSize": "18px",
              "marginBottom": "40px",
              "marginTop": "0px",
            }
          }
        >
          Web Jam LLC helps businesses bring their web technologies into the 21st century! Let us help make your website mobile-friendly and grow your business by utilizing the power of online services. More information is available in our 
          <a
            href="https://www.dropbox.com/s/bzdqp3hr682y9sw/WebJamLLC_FactSheet.pdf?dl=0"
            rel="noopener noreferrer"
            target="_blank"
          >
            Fact Sheet
          </a>
          .
        </p>
        <div
          className="faqs"
        >
          <h3>
            FAQs
          </h3>
          <div
            className="qAnda"
          >
            <p>
              <span>
                Q: 
              </span>
              How can I contact you?
            </p>
            <p>
              <span>
                A: 
              </span>
              Send an email to Josh & Maria Sherman
              <br />
              <a
                href="mailto:web.jam.adm@gmail.com"
              >
                <strong>
                  web.jam.adm@gmail.com
                </strong>
              </a>
              <br />
              or call 
              <a
                href="tel:5404948035"
              >
                <strong>
                  540-494-8035
                </strong>
              </a>
            </p>
            <br />
            <p>
              <span>
                Q: 
              </span>
              What does Web Jam LLC do?
            </p>
            <p>
              <span>
                A: 
              </span>
              We provide mobile-friendly web technologies and services.
            </p>
            <br />
            <p>
              <span>
                Q: 
              </span>
              What is your mission statement?
            </p>
            <p>
              <span>
                A: 
              </span>
              To assist small businesses by refactoring their existing websites to be mobile-friendly, providing new online services, empowering business owners, and training staff to use these new services effectively.
            </p>
          </div>
        </div>
      </div>
    </div>
    <hr />
    <div
      className="notWidescreen"
    >
      <div
        className="material-content"
      >
        <h3
          style={
            {
              "marginBottom": "15px",
              "marginTop": "15px",
              "textAlign": "center",
            }
          }
        >
          Our Current Projects
        </h3>
        <div
          className="elevation2 project"
          style={
            {
              "minWidth": "313px",
            }
          }
        >
          <h4>
            College Lutheran Church
          </h4>
          <img
            alt="College Lutheran Church Homepage"
            src="https://dl.dropboxusercontent.com/s/354maifx0dkzt9s/Screenshot%20from%202019-02-22%2016-01-50.png?dl=0"
          />
          <br />
          <a
            href="https://www.collegelutheran.org"
            rel="noreferrer noopener"
            target="_blank"
          >
            Collegelutheran.org
          </a>
        </div>
        <p
          style={
            {
              "fontSize": "6pt",
              "marginBottom": "0",
            }
          }
        >
           
        </p>
        <div
          className="elevation2 project apperson"
        >
          <h4>
            Apperson Automotive
          </h4>
          <div>
            <img
              alt="Apperson Automotive Home Page"
              src="../static/imgs/AppersonAutomotive.png"
            />
            <br />
            <a
              href="https://www.appersonautomotive.com"
              rel="noreferrer noopener"
              target="_blank"
            >
              appersonautomotive.com
            </a>
          </div>
        </div>
      </div>
    </div>
    <p
      style={
        {
          "fontSize": "6pt",
          "marginBottom": "0",
        }
      }
    >
       
    </p>
    <div
      className="notWidescreen"
      style={
        {
          "margin": "auto",
          "maxWidth": "300px",
        }
      }
    >
      <iframe
        allow="encrypted-media"
        className="notWideScreen"
        height="300"
        scrolling="none"
        src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2FWebJamLLC%2F&tabs=timeline&width=300&height=500&small_header=true&adapt_container_width=true&hide_cover=false&show_facepile=true&appId=2207148322688942"
        style={
          {
            "border": "none",
            "overflow": "hidden",
          }
        }
        title="facebook ticker"
        width="300"
      />
    </div>
    <p
      style={
        {
          "fontSize": "6pt",
          "marginBottom": "0",
        }
      }
    >
       
    </p>
    <div
      className="row form-row"
      style={
        {
          "border": "1px solid black",
          "margin": "auto",
        }
      }
    >
      <div
        className="contact-form"
      >
        <h4
          style={
            {
              "fontWeight": "bold",
              "marginBottom": "0",
              "marginTop": "10px",
              "paddingTop": 0,
              "textAlign": "center",
            }
          }
        >
          Contact Us
        </h4>
        <form
          className="col"
          id="new-contact"
        >
          <table
            style={
              {
                "border": "none",
                "margin": 0,
                "marginBottom": "20px",
                "padding": 0,
                "textAlign": "left",
              }
            }
          >
            <tbody>
              <tr
                className="white-background"
              >
                <td
                  style={
                    {
                      "border": "none",
                      "padding": 0,
                    }
                  }
                >
                  <input
                    id="firstname"
                    name="firstname"
                    onChange={[Function]}
                    placeholder="* First Name"
                    type="text"
                    value=""
                  />
                </td>
                <td
                  style={
                    {
                      "border": "none",
                      "padding": "8px",
                    }
                  }
                >
                   
                </td>
                <td
                  style={
                    {
                      "border": "none",
                      "padding": 0,
                    }
                  }
                >
                  <input
                    id="lastname"
                    name="lastname"
                    onChange={[Function]}
                    placeholder="* Last Name"
                    type="text"
                    value=""
                  />
                </td>
              </tr>
              <tr
                className="white-background"
              >
                <td
                  style={
                    {
                      "border": "none",
                      "padding": "8px",
                    }
                  }
                >
                   
                </td>
              </tr>
              <tr
                className="white-background"
              >
                <td
                  style={
                    {
                      "border": "none",
                      "padding": 0,
                    }
                  }
                >
                  <input
                    id="emailaddress"
                    name="emailaddress"
                    onChange={[Function]}
                    placeholder="* Email Address"
                    type="email"
                    value=""
                  />
                </td>
                <td
                  style={
                    {
                      "border": "none",
                      "padding": "8px",
                    }
                  }
                >
                   
                </td>
                <td
                  className="phone"
                  style={
                    {
                      "border": "none",
                      "padding": 0,
                    }
                  }
                >
                   
                  <input
                    id="phonenumber"
                    name="phonenumber"
                    onChange={[Function]}
                    placeholder="* Phone Number"
                    type="tel"
                    value=""
                  />
                </td>
              </tr>
            </tbody>
          </table>
          <select
            id="country"
            multiple={false}
            onChange={[Function]}
            style={
              {
                "width": "100%",
              }
            }
          >
            <option
              id="* Select Your Country"
              value="* Select Your Country"
            >
              * Select Your Country
            </option>
            <option
              id="Afghanistan"
              value="Afghanistan"
            >
              Afghanistan
            </option>
            <option
              id="Albania"
              value="Albania"
            >
              Albania
            </option>
            <option
              id="Algeria"
              value="Algeria"
            >
              Algeria
            </option>
            <option
              id="Andorra"
              value="Andorra"
            >
              Andorra
            </option>
            <option
              id="Angola"
              value="Angola"
            >
              Angola
            </option>
            <option
              id="Antigua & Deps"
              value="Antigua & Deps"
            >
              Antigua & Deps
            </option>
            <option
              id="Argentina"
              value="Argentina"
            >
              Argentina
            </option>
            <option
              id="Armenia"
              value="Armenia"
            >
              Armenia
            </option>
            <option
              id="Australia"
              value="Australia"
            >
              Australia
            </option>
            <option
              id="Austria"
              value="Austria"
            >
              Austria
            </option>
            <option
              id="Azerbaijan"
              value="Azerbaijan"
            >
              Azerbaijan
            </option>
            <option
              id="Bahamas"
              value="Bahamas"
            >
              Bahamas
            </option>
            <option
              id="Bahrain"
              value="Bahrain"
            >
              Bahrain
            </option>
            <option
              id="Bangladesh"
              value="Bangladesh"
            >
              Bangladesh
            </option>
            <option
              id="Barbados"
              value="Barbados"
            >
              Barbados
            </option>
            <option
              id="Belarus"
              value="Belarus"
            >
              Belarus
            </option>
            <option
              id="Belgium"
              value="Belgium"
            >
              Belgium
            </option>
            <option
              id="Belize"
              value="Belize"
            >
              Belize
            </option>
            <option
              id="Benin"
              value="Benin"
            >
              Benin
            </option>
            <option
              id="Bhutan"
              value="Bhutan"
            >
              Bhutan
            </option>
            <option
              id="Bolivia"
              value="Bolivia"
            >
              Bolivia
            </option>
            <option
              id="Bosnia Herzegovina"
              value="Bosnia Herzegovina"
            >
              Bosnia Herzegovina
            </option>
            <option
              id="Botswana"
              value="Botswana"
            >
              Botswana
            </option>
            <option
              id="Brazil"
              value="Brazil"
            >
              Brazil
            </option>
            <option
              id="Brunei"
              value="Brunei"
            >
              Brunei
            </option>
            <option
              id="Bulgaria"
              value="Bulgaria"
            >
              Bulgaria
            </option>
            <option
              id="Burkina"
              value="Burkina"
            >
              Burkina
            </option>
            <option
              id="Burundi"
              value="Burundi"
            >
              Burundi
            </option>
            <option
              id="Cambodia"
              value="Cambodia"
            >
              Cambodia
            </option>
            <option
              id="Cameroon"
              value="Cameroon"
            >
              Cameroon
            </option>
            <option
              id="Canada"
              value="Canada"
            >
              Canada
            </option>
            <option
              id="Cape Verde"
              value="Cape Verde"
            >
              Cape Verde
            </option>
            <option
              id="Central African Rep"
              value="Central African Rep"
            >
              Central African Rep
            </option>
            <option
              id="Chad"
              value="Chad"
            >
              Chad
            </option>
            <option
              id="Chile"
              value="Chile"
            >
              Chile
            </option>
            <option
              id="China"
              value="China"
            >
              China
            </option>
            <option
              id="Colombia"
              value="Colombia"
            >
              Colombia
            </option>
            <option
              id="Comoros"
              value="Comoros"
            >
              Comoros
            </option>
            <option
              id="Congo"
              value="Congo"
            >
              Congo
            </option>
            <option
              id="Congo (Democratic Rep)"
              value="Congo (Democratic Rep)"
            >
              Congo (Democratic Rep)
            </option>
            <option
              id="Costa Rica"
              value="Costa Rica"
            >
              Costa Rica
            </option>
            <option
              id="Croatia"
              value="Croatia"
            >
              Croatia
            </option>
            <option
              id="Cuba"
              value="Cuba"
            >
              Cuba
            </option>
            <option
              id="Cyprus"
              value="Cyprus"
            >
              Cyprus
            </option>
            <option
              id="Czech Republic"
              value="Czech Republic"
            >
              Czech Republic
            </option>
            <option
              id="Denmark"
              value="Denmark"
            >
              Denmark
            </option>
            <option
              id="Djibouti"
              value="Djibouti"
            >
              Djibouti
            </option>
            <option
              id="Dominica"
              value="Dominica"
            >
              Dominica
            </option>
            <option
              id="Dominican Republic"
              value="Dominican Republic"
            >
              Dominican Republic
            </option>
            <option
              id="East Timor"
              value="East Timor"
            >
              East Timor
            </option>
            <option
              id="Ecuador"
              value="Ecuador"
            >
              Ecuador
            </option>
            <option
              id="Egypt"
              value="Egypt"
            >
              Egypt
            </option>
            <option
              id="El Salvador"
              value="El Salvador"
            >
              El Salvador
            </option>
            <option
              id="Equatorial Guinea"
              value="Equatorial Guinea"
            >
              Equatorial Guinea
            </option>
            <option
              id="Eritrea"
              value="Eritrea"
            >
              Eritrea
            </option>
            <option
              id="Estonia"
              value="Estonia"
            >
              Estonia
            </option>
            <option
              id="Ethiopia"
              value="Ethiopia"
            >
              Ethiopia
            </option>
            <option
              id="Fiji"
              value="Fiji"
            >
              Fiji
            </option>
            <option
              id="Finland"
              value="Finland"
            >
              Finland
            </option>
            <option
              id="France"
              value="France"
            >
              France
            </option>
            <option
              id="Gabon"
              value="Gabon"
            >
              Gabon
            </option>
            <option
              id="Gambia"
              value="Gambia"
            >
              Gambia
            </option>
            <option
              id="Georgia"
              value="Georgia"
            >
              Georgia
            </option>
            <option
              id="Germany"
              value="Germany"
            >
              Germany
            </option>
            <option
              id="Ghana"
              value="Ghana"
            >
              Ghana
            </option>
            <option
              id="Greece"
              value="Greece"
            >
              Greece
            </option>
            <option
              id="Grenada"
              value="Grenada"
            >
              Grenada
            </option>
            <option
              id="Guatemala"
              value="Guatemala"
            >
              Guatemala
            </option>
            <option
              id="Guinea"
              value="Guinea"
            >
              Guinea
            </option>
            <option
              id="Guinea-Bissau"
              value="Guinea-Bissau"
            >
              Guinea-Bissau
            </option>
            <option
              id="Guyana"
              value="Guyana"
            >
              Guyana
            </option>
            <option
              id="Haiti"
              value="Haiti"
            >
              Haiti
            </option>
            <option
              id="Honduras"
              value="Honduras"
            >
              Honduras
            </option>
            <option
              id="Hungary"
              value="Hungary"
            >
              Hungary
            </option>
            <option
              id="Iceland"
              value="Iceland"
            >
              Iceland
            </option>
            <option
              id="India"
              value="India"
            >
              India
            </option>
            <option
              id="Indonesia"
              value="Indonesia"
            >
              Indonesia
            </option>
            <option
              id="Iran"
              value="Iran"
            >
              Iran
            </option>
            <option
              id="Iraq"
              value="Iraq"
            >
              Iraq
            </option>
            <option
              id="Ireland (Republic)"
              value="Ireland (Republic)"
            >
              Ireland (Republic)
            </option>
            <option
              id="Israel"
              value="Israel"
            >
              Israel
            </option>
            <option
              id="Italy"
              value="Italy"
            >
              Italy
            </option>
            <option
              id="Ivory Coast"
              value="Ivory Coast"
            >
              Ivory Coast
            </option>
            <option
              id="Jamaica"
              value="Jamaica"
            >
              Jamaica
            </option>
            <option
              id="Japan"
              value="Japan"
            >
              Japan
            </option>
            <option
              id="Jordan"
              value="Jordan"
            >
              Jordan
            </option>
            <option
              id="Kazakhstan"
              value="Kazakhstan"
            >
              Kazakhstan
            </option>
            <option
              id="Kenya"
              value="Kenya"
            >
              Kenya
            </option>
            <option
              id="Kiribati"
              value="Kiribati"
            >
              Kiribati
            </option>
            <option
              id="Kosovo"
              value="Kosovo"
            >
              Kosovo
            </option>
            <option
              id="Kuwait"
              value="Kuwait"
            >
              Kuwait
            </option>
            <option
              id="Kyrgyzstan"
              value="Kyrgyzstan"
            >
              Kyrgyzstan
            </option>
            <option
              id="Laos"
              value="Laos"
            >
              Laos
            </option>
            <option
              id="Latvia"
              value="Latvia"
            >
              Latvia
            </option>
            <option
              id="Lebanon"
              value="Lebanon"
            >
              Lebanon
            </option>
            <option
              id="Lesotho"
              value="Lesotho"
            >
              Lesotho
            </option>
            <option
              id="Liberia"
              value="Liberia"
            >
              Liberia
            </option>
            <option
              id="Libya"
              value="Libya"
            >
              Libya
            </option>
            <option
              id="Liechtenstein"
              value="Liechtenstein"
            >
              Liechtenstein
            </option>
            <option
              id="Lithuania"
              value="Lithuania"
            >
              Lithuania
            </option>
            <option
              id="Luxembourg"
              value="Luxembourg"
            >
              Luxembourg
            </option>
            <option
              id="Macedonia"
              value="Macedonia"
            >
              Macedonia
            </option>
            <option
              id="Madagascar"
              value="Madagascar"
            >
              Madagascar
            </option>
            <option
              id="Malawi"
              value="Malawi"
            >
              Malawi
            </option>
            <option
              id="Malaysia"
              value="Malaysia"
            >
              Malaysia
            </option>
            <option
              id="Maldives"
              value="Maldives"
            >
              Maldives
            </option>
            <option
              id="Mali"
              value="Mali"
            >
              Mali
            </option>
            <option
              id="Malta"
              value="Malta"
            >
              Malta
            </option>
            <option
              id="Marshall Islands"
              value="Marshall Islands"
            >
              Marshall Islands
            </option>
            <option
              id="Mauritania"
              value="Mauritania"
            >
              Mauritania
            </option>
            <option
              id="Mauritius"
              value="Mauritius"
            >
              Mauritius
            </option>
            <option
              id="Mexico"
              value="Mexico"
            >
              Mexico
            </option>
            <option
              id="Micronesia"
              value="Micronesia"
            >
              Micronesia
            </option>
            <option
              id="Moldova"
              value="Moldova"
            >
              Moldova
            </option>
            <option
              id="Monaco"
              value="Monaco"
            >
              Monaco
            </option>
            <option
              id="Mongolia"
              value="Mongolia"
            >
              Mongolia
            </option>
            <option
              id="Montenegro"
              value="Montenegro"
            >
              Montenegro
            </option>
            <option
              id="Morocco"
              value="Morocco"
            >
              Morocco
            </option>
            <option
              id="Mozambique"
              value="Mozambique"
            >
              Mozambique
            </option>
            <option
              id="Myanmar, (Burma)"
              value="Myanmar, (Burma)"
            >
              Myanmar, (Burma)
            </option>
            <option
              id="Namibia"
              value="Namibia"
            >
              Namibia
            </option>
            <option
              id="Nauru"
              value="Nauru"
            >
              Nauru
            </option>
            <option
              id="Nepal"
              value="Nepal"
            >
              Nepal
            </option>
            <option
              id="Netherlands"
              value="Netherlands"
            >
              Netherlands
            </option>
            <option
              id="New Zealand"
              value="New Zealand"
            >
              New Zealand
            </option>
            <option
              id="Nicaragua"
              value="Nicaragua"
            >
              Nicaragua
            </option>
            <option
              id="Niger"
              value="Niger"
            >
              Niger
            </option>
            <option
              id="Nigeria"
              value="Nigeria"
            >
              Nigeria
            </option>
            <option
              id="North Korea"
              value="North Korea"
            >
              North Korea
            </option>
            <option
              id="Norway"
              value="Norway"
            >
              Norway
            </option>
            <option
              id="Oman"
              value="Oman"
            >
              Oman
            </option>
            <option
              id="Pakistan"
              value="Pakistan"
            >
              Pakistan
            </option>
            <option
              id="Palau"
              value="Palau"
            >
              Palau
            </option>
            <option
              id="Panama"
              value="Panama"
            >
              Panama
            </option>
            <option
              id="Papua New Guinea"
              value="Papua New Guinea"
            >
              Papua New Guinea
            </option>
            <option
              id="Paraguay"
              value="Paraguay"
            >
              Paraguay
            </option>
            <option
              id="Peru"
              value="Peru"
            >
              Peru
            </option>
            <option
              id="Philippines"
              value="Philippines"
            >
              Philippines
            </option>
            <option
              id="Poland"
              value="Poland"
            >
              Poland
            </option>
            <option
              id="Portugal"
              value="Portugal"
            >
              Portugal
            </option>
            <option
              id="Qatar"
              value="Qatar"
            >
              Qatar
            </option>
            <option
              id="Romania"
              value="Romania"
            >
              Romania
            </option>
            <option
              id="Russian Federation"
              value="Russian Federation"
            >
              Russian Federation
            </option>
            <option
              id="Rwanda"
              value="Rwanda"
            >
              Rwanda
            </option>
            <option
              id="Saint Vincent & the Grenadines"
              value="Saint Vincent & the Grenadines"
            >
              Saint Vincent & the Grenadines
            </option>
            <option
              id="Samoa"
              value="Samoa"
            >
              Samoa
            </option>
            <option
              id="San Marino"
              value="San Marino"
            >
              San Marino
            </option>
            <option
              id="Sao Tome & Principe"
              value="Sao Tome & Principe"
            >
              Sao Tome & Principe
            </option>
            <option
              id="Saudi Arabia"
              value="Saudi Arabia"
            >
              Saudi Arabia
            </option>
            <option
              id="Senegal"
              value="Senegal"
            >
              Senegal
            </option>
            <option
              id="Serbia"
              value="Serbia"
            >
              Serbia
            </option>
            <option
              id="Seychelles"
              value="Seychelles"
            >
              Seychelles
            </option>
            <option
              id="Sierra Leone"
              value="Sierra Leone"
            >
              Sierra Leone
            </option>
            <option
              id="Singapore"
              value="Singapore"
            >
              Singapore
            </option>
            <option
              id="Slovakia"
              value="Slovakia"
            >
              Slovakia
            </option>
            <option
              id="Slovenia"
              value="Slovenia"
            >
              Slovenia
            </option>
            <option
              id="Solomon Islands"
              value="Solomon Islands"
            >
              Solomon Islands
            </option>
            <option
              id="Somalia"
              value="Somalia"
            >
              Somalia
            </option>
            <option
              id="South Africa"
              value="South Africa"
            >
              South Africa
            </option>
            <option
              id="South Korea"
              value="South Korea"
            >
              South Korea
            </option>
            <option
              id="South Sudan"
              value="South Sudan"
            >
              South Sudan
            </option>
            <option
              id="Spain"
              value="Spain"
            >
              Spain
            </option>
            <option
              id="Sri Lanka"
              value="Sri Lanka"
            >
              Sri Lanka
            </option>
            <option
              id="St Kitts & Nevis"
              value="St Kitts & Nevis"
            >
              St Kitts & Nevis
            </option>
            <option
              id="St Lucia"
              value="St Lucia"
            >
              St Lucia
            </option>
            <option
              id="Sudan"
              value="Sudan"
            >
              Sudan
            </option>
            <option
              id="Suriname"
              value="Suriname"
            >
              Suriname
            </option>
            <option
              id="Swaziland"
              value="Swaziland"
            >
              Swaziland
            </option>
            <option
              id="Sweden"
              value="Sweden"
            >
              Sweden
            </option>
            <option
              id="Switzerland"
              value="Switzerland"
            >
              Switzerland
            </option>
            <option
              id="Syria"
              value="Syria"
            >
              Syria
            </option>
            <option
              id="Taiwan"
              value="Taiwan"
            >
              Taiwan
            </option>
            <option
              id="Tajikistan"
              value="Tajikistan"
            >
              Tajikistan
            </option>
            <option
              id="Tanzania"
              value="Tanzania"
            >
              Tanzania
            </option>
            <option
              id="Thailand"
              value="Thailand"
            >
              Thailand
            </option>
            <option
              id="Togo"
              value="Togo"
            >
              Togo
            </option>
            <option
              id="Tonga"
              value="Tonga"
            >
              Tonga
            </option>
            <option
              id="Trinidad & Tobago"
              value="Trinidad & Tobago"
            >
              Trinidad & Tobago
            </option>
            <option
              id="Tunisia"
              value="Tunisia"
            >
              Tunisia
            </option>
            <option
              id="Turkey"
              value="Turkey"
            >
              Turkey
            </option>
            <option
              id="Turkmenistan"
              value="Turkmenistan"
            >
              Turkmenistan
            </option>
            <option
              id="Tuvalu"
              value="Tuvalu"
            >
              Tuvalu
            </option>
            <option
              id="Uganda"
              value="Uganda"
            >
              Uganda
            </option>
            <option
              id="Ukraine"
              value="Ukraine"
            >
              Ukraine
            </option>
            <option
              id="United Arab Emirates"
              value="United Arab Emirates"
            >
              United Arab Emirates
            </option>
            <option
              id="United Kingdom"
              value="United Kingdom"
            >
              United Kingdom
            </option>
            <option
              id="United States"
              value="United States"
            >
              United States
            </option>
            <option
              id="Uruguay"
              value="Uruguay"
            >
              Uruguay
            </option>
            <option
              id="Uzbekistan"
              value="Uzbekistan"
            >
              Uzbekistan
            </option>
            <option
              id="Vanuatu"
              value="Vanuatu"
            >
              Vanuatu
            </option>
            <option
              id="Vatican City"
              value="Vatican City"
            >
              Vatican City
            </option>
            <option
              id="Venezuela"
              value="Venezuela"
            >
              Venezuela
            </option>
            <option
              id="Vietnam"
              value="Vietnam"
            >
              Vietnam
            </option>
            <option
              id="Yemen"
              value="Yemen"
            >
              Yemen
            </option>
            <option
              id="Zambia"
              value="Zambia"
            >
              Zambia
            </option>
            <option
              id="Zimbabwe"
              value="Zimbabwe"
            >
              Zimbabwe
            </option>
          </select>
           
          <p
            style={
              {
                "margin": 0,
              }
            }
          >
             
          </p>
          <input
            id="zipcode"
            name="zipcode"
            onChange={[Function]}
            placeholder="* Zipcode"
            style={
              {
                "width": "100%",
              }
            }
            type="text"
            value=""
          />
          <p
            style={
              {
                "margin": 0,
              }
            }
          >
             
          </p>
          <input
            className="comments"
            onChange={[Function]}
            placeholder="Comments"
            style={
              {
                "height": "80px",
                "marginTop": "20px",
                "width": "100%",
              }
            }
          />
          <div
            className="inquiryValidation input-field col"
            style={
              {
                "marginBottom": "12px",
              }
            }
          >
            <span
              className="inquiryValidation"
            >
              * Required
            </span>
            <button
              disabled={true}
              id="sendEmailButton"
              onClick={[Function]}
            >
              Submit
            </button>
          </div>
        </form>
      </div>
    </div>
    <p
      style={
        {
          "fontSize": "6pt",
          "marginBottom": "0",
        }
      }
    >
       
    </p>
  </div>
</div>
`;
